app/src/main/java/com/fstyle/androidtrainning/data/local/storage/External.java

```diff
@@ -0,0 +1,311 @@
+package com.fstyle.androidtrainning.data.local.storage;
+
+import android.content.ContentResolver;
+import android.content.Context;
+import android.database.Cursor;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.net.Uri;
+import android.provider.MediaStore;
+import android.util.Log;
```

Comment:
ERROR: Unused import: android.util.Log. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports